app/cdap/components/FooterPanel/IconStore/ColumnIcon.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const ColumnIcon = (
```

Comment:
I don't think this icon is necessary - please use material ui icons for simple things like this. For instance: https://mui.com/material-ui/material-icons/?query=menu&selected=Menu 

if that one isn't close enough feel free to use a different one that you find there.

Reply:
This was discussed between Arunima(Google) & Vinay(Divami). It will be delivered as a part of version 3.